contrib/builder/deb/armhf/debian-jessie/Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM armhf/debian:jessie
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/deb/armhf/generate.sh"!
+#
 
-# allow replacing httpredir mirror
-ARG APT_MIRROR=httpredir.debian.org
-RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
+FROM debian:jessie
```

Comment:
I don't think this will work when this is run on an arm machine. Gotta have `armhf/debian:jessie`